src/main/java/appeng/container/implementations/ContainerUpgradeable.java

```diff
@@ -48,6 +41,13 @@
 import appeng.items.tools.ToolNetworkTool;
 import appeng.parts.automation.PartExportBus;
 import appeng.util.Platform;
+import net.minecraft.entity.player.InventoryPlayer;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Wrong order for 'net.minecraft.entity.player.InventoryPlayer' import. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck)